lib/client/index.js

```diff
@@ -49,6 +49,7 @@ module.exports = ({ port = null, config = {}, filters = {} }) => {
       config.brokerClientValidationMethod || 'GET';
     const brokerClientValidationTimeoutMs =
       config.brokerClientValidationTimeoutMs || 5000;
+    const isJsonResponse = !config.brokerClientValidationJsonDisabled;
```

Comment:
I'd call it `...ENABLED` then, and not `...DISABLED`. Right? 🤔

Reply:
The point is to turn it off only for artifactory integration. If we will do `_ENABLED`, we need to update all .env files for all integrations...